doc/harmony-ap_installation_guide.md

```diff
@@ -80,10 +80,10 @@ See eDelivery documentation \[[TERMS](#Ref_TERMS)\].
 
 ### 2.1 Prerequisites to Installation
 
-The Access Point is officially supported on the following platforms:
+The Access Point is officially supported on the following operating system:
```

Comment:
This should be plural and I would also mention the supported architectures => "operating systems (`x86-64`, `arm64`)".